ktor-client/ktor-client-plugins/ktor-client-content-negotiation/common/src/io/ktor/client/plugins/JsonContentTypeMatcher.kt

```diff
@@ -1,21 +0,0 @@
-/*
```

Comment:
Please keep this class for backwards compatibility.

Reply:
But it was never released, was it? ContentNegotiation feature was only added in 2.0.0

Reply:
Happy to add it back, just want to make sure you don;t take up on unnecessary deprecation cycle.

Reply:
you are right, thanks for correcting me